test/operator-fan-out.spec.ts

```diff
@@ -0,0 +1,60 @@
+import 'mocha';
+import { expect } from 'chai';
+import deepcopy from 'deepcopy';
+
+import { DataLayerObserver } from '../src/observer';
+
+const item = {
+  'left-hand': { fingers: { index: true, thumb: false } },
+  'right-hand': { fingers: { middle: false, pinky: true } },
+  list: [
+    { eenie: 'meenie' },
+    { minie: 'moe' },
+  ],
+};
+
+// @ts-ignore
+function handleDestination(...params: any[]) {
+  (globalThis as any).callParameters.push(params);
+}
+
+describe('fan out operator unit tests', () => {
+  beforeEach(() => {
+    (globalThis as any).item = deepcopy(item);
+    (globalThis as any).handleDestination = handleDestination;
+    (globalThis as any).callParameters = [];
+  });
+
+  afterEach(() => {
+    delete (globalThis as any).item;
+    delete (globalThis as any).callParameters;
+  });
+
+  it('it should fan out properties', () => {
+    const observer = new DataLayerObserver({
+      rules: [
+        {
+          source: 'item',
+          operators: [
+            { name: 'fan-out', properties: ['left-hand', 'right-hand', 'list'] },
+            { name: 'flatten' },
+          ],
+          destination: 'globalThis.handleDestination',
```

Comment:
`destination` will actually accept a function too.  In the observer spec, I have:

```
let changes = [];
...
destination: (...data: any[]) => {
            // NOTE use a local destination to prevent cross-test pollution
            changes = data;
          },
```

Nothing functional incorrect, just in case you wanted to keep the destination and result localized.